src/Client/Invoice.php

```diff
@@ -70,6 +70,43 @@ public function createInvoice(
         }
     }
 
+    public function updateInvoice(
+        string $storeId,
+        string $invoiceId,
+        ?array $metaData = null
+    ): ResultInvoice {
+        $url = $this->getApiUrl() . 'stores/' . urlencode(
+            $storeId
+        ) . '/invoices/' . urlencode($invoiceId);
+        $headers = $this->getRequestHeaders();
+        $method = 'PUT';
+
+        // Prepare metadata.
```

Comment:
you can remove all of this and in line 94, instead of: 

`'metadata' => !empty($metaDataMerged) ? $metaDataMerged : null`

change to:

`'metadata' => $metaData`